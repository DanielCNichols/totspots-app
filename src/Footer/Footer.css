.footer {
  width: 100%;
  height: 30px;
  display: block;
  margin: 0;
  padding: 0;
}

.footer p {
  margin: 0;
}
