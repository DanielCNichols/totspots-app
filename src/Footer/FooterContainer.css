.FooterContainer {
  position:absolute;
  left:0;
  bottom:0;
  right:0;
  height: 70px;
  margin: 0;
  padding: 0;
  text-align: center;
  display: flex;
  flex-direction: column;
  align-items: center;
  justify-content: center;
}
