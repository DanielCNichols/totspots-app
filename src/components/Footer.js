import React from 'react';
import s from '../sass/components/Footer.module.scss';

export default function Footer() {
  return (
    <footer className={s.footer}>
      <p>2020 TotSpots</p>
    </footer>
  );
}
