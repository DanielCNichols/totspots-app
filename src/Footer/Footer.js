import React from 'react'
import './Footer.css'


export default function Footer() {

  return (
    <div className="footer">
       <p>Daniel Nichols | 2019</p>
       <a href="https://github.com/DanielCNichols"><p>Want to see more? Find it on GitHub</p></a>
    </div>
  )
}