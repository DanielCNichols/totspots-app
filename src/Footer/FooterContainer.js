import React from  'react';
import Footer from './Footer'
import './FooterContainer.css'


export default function FooterContainer() {

  return (
    <div className="FooterContainer">
      <Footer></Footer>
    </div>
  )
}