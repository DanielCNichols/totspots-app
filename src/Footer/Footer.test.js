import React from 'react';
import ReactDOM from 'react-dom';
import Footer from './Footer.js';
import renderer from 'react-test-renderer'

it('Footer renders without crashing', () => {
  const div = document.createElement('div');
  ReactDOM.render(<Footer />, div);
  ReactDOM.unmountComponentAtNode(div);
});

it('renders Footer as expected', () => {
  const tree = renderer.create(<Footer name='Footer' />).toJSON();
  expect(tree).toMatchSnapshot();
});
