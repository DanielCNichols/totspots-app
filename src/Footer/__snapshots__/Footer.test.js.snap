// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders Footer as expected 1`] = `
<div
  className="footer"
>
  <p>
    Daniel Nichols | 2019
  </p>
  <p>
    Want to see more? Find it on GitHub
  </p>
</div>
`;
